compile.js

```diff
@@ -6,10 +6,14 @@ fs.readFile("./emojidata/emoji.json", "utf8", (err, data) => {
   const emojis = JSON.parse(data);
   for (const emoji of emojis) {
     const newEmoji = {
-      short_name: emoji.short_name,
-      image: emoji.image,
-      text: emoji.text
+      image: emoji.image
     };
+    if( emoji.texts) {
+      newEmoji.texts = emoji.texts
+    }
+    if (emoji.short_names) {
+      newEmoji.short_names = emoji.short_names
+    }
```

Comment:
Include all short names and texts in the minified JSON file. One emoji has texts `[":o",":-o",":O",":-O"]` for example.